src/core/settings.h

```diff
@@ -98,6 +99,7 @@ typedef struct QUIC_SETTINGS_INTERNAL {
     uint8_t EcnEnabled                      : 1;
     uint8_t HyStartEnabled                  : 1;
     uint8_t MtuDiscoveryMissingProbeCount;
+    uint8_t EncryptionOffloadEnabled        : 1;
```

Comment:
```suggestion
    uint8_t EncryptionOffloadEnabled        : 1;
    uint8_t MtuDiscoveryMissingProbeCount;
```
The core struct can be reordered. Let's keep the bitfields together.